methods/transient/Transient_approach2.1.ipynb

```diff
@@ -0,0 +1,271 @@
+{
```

Comment:
<p>This one also needs some introduction about what this approach is doing, including the fact that user has to run the&nbsp;<code>Transient_base</code>&nbsp;notebook first.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nisar-solid/ATBD/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>